step-templates/wait.json

```diff
@@ -3,11 +3,11 @@
   "Name": "Wait",
   "Description": "Pauses the process for a set duration",
   "ActionType": "Octopus.Script",
-  "Version": 1,
+  "Version": 2,
   "CommunityActionTemplateId": null,
   "Packages": [],
   "Properties": {
-    "Octopus.Action.Script.ScriptBody": "$Wait = $Seconds\nfor ($CountDown = $Wait; $CountDown -ge 0; $CountDown--){\nWrite-Verbose \"$CountDown seconds remaining\"\nStart-Sleep 1\n}",
+    "Octopus.Action.Script.ScriptBody": "[int]$Wait = $Seconds\nWrite-Output "Waiting for $seconds seconds"\nfor ($CountDown = $Wait; $CountDown -ge 0; $CountDown--){\nWrite-Verbose \"$CountDown seconds remaining\"\nStart-Sleep 1\n}",
```

Comment:
```suggestion
    "Octopus.Action.Script.ScriptBody": "[int]$Wait = $Seconds\nWrite-Output \"Waiting for $seconds seconds\"\nfor ($CountDown = $Wait; $CountDown -ge 0; $CountDown--){\nWrite-Verbose \"$CountDown seconds remaining\"\nStart-Sleep 1\n}",
```

Reply:
@Biohazardhpk We need to escape the "s here. If you're good with the change, we can commit and merge the PR.

Reply:
Hi @Biohazardhpk, 

I wanted to check in and see if this change is ok to be made and we can publish the step?

Thanks